js/main.js

```diff
@@ -1,2 +1,162 @@
 $(function () {
-})
+
+	let botChoice
+	let userChoice
+	let botScore = 0
+	let userScore = 0
+	let botWins = 0
+	let userWins = 0
+	resetGame()
+
+	$('button').click((event) => {
+
+		//Calling all functions
+		clearBonusMessage()
+		assignBotChoice()
+		assignUserChoice()
+		determineWinner()
+		bonusPoints()
+		roundOneWinner()
+		roundTwoWinner()
+	})
+
+	//Randomly assigning a choice to the bot
+	function assignBotChoice() {
+		//Random number from 1 to 3
+		botChoice = Math.floor(Math.random() * 3) + 1
+
+		//If statement to assign numbers to string choices
+		if (botChoice === 1) {
+			botChoice = 'rock'
+		} else if (botChoice === 2) {
+			botChoice = 'scissors'
+		} else if (botChoice === 3) {
+			botChoice = 'paper'
+		} else {
+			console.log('Invalid choice.')
+		}
+	}
+
+	//Grabbing the clicked button id and converting that choice to a string
+	function assignUserChoice() {
+		userChoice = $(event.target).attr('id')
+	}
+
+	//Function to compare choices, declare a winner, and reward points
+	function determineWinner() {
+		if ( userChoice === botChoice ) {
+			$('#status').html(`<p>You chose <b><u>${userChoice}</u></b>. The bot chose <b><u>${botChoice}</u></b>.</p><p>Tie game!</p>`)
```

Comment:
@cswebdesign the only part that changes in the status is the `userChoice` and `botChoice`. Because the string is used several times, you may want to consider encapsulating it into a function.

[Here](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/master/js/main.js#L74) is the alternative approach